### ISSUE TITLE
strapi is not recognised in command line

### ISSUE THREAD

**Issue by EDMONDGIHOZO** [strapi/strapi#1901]
**Informations**
- **Node.js version**:
- **npm version**:
- **Strapi version**:
- **Database**:
- **Operating system**:

**What is the current behavior?**

**Steps to reproduce the problem**

**What is the expected behavior?**

**Suggested solutions**

**Comment by derrickmehaffy** [comment]
@EDMONDGIHOZO please fillout the bug report, lack of required information will cause this issue to be closed.

**Comment by EDMONDGIHOZO** [comment]
after installing strapi by npm , i tried to create project by the command line says that 'strapi' is not recognized as an internal or external command,
operable program or batch file.

**Comment by maturanomx** [comment]
- Edit your original message, fill the required information as requested.
- Provide enough information to help you. Show us all the process, what was the output of the installation?

Probably is not an strapi problem. Did you try with other package?... let's say... try installing `nls`; do you have the same error?

**Comment by derrickmehaffy** [comment]
@EDMONDGIHOZO If I had to guess you are on Windows which has limited support. The error is likely due to npm global packages not mapped to your $PATH.

**Comment by arjohnson97** [comment]
I have the same issue, putting npx in front of it works for me.

**Comment by lauriejim** [comment]
Hello @EDMONDGIHOZO , please follow the issue template so we can get a better understanding about the origin of your bug.
I will reopen your issue as soon as it follows the templating guidelines.
Thank you.

**Comment by shrutisahatpure** [comment]
Hi, @EDMONDGIHOZO. I had same issue then I tried to Install strapi globally by 'npm install -g strapi@beta' . This solved my issue. Hope this will help you :)

**Comment by jeffpohlmeyer** [comment]
I'm going to go ahead and add to this.
Node: 12.17.0
NPM: 6.14.4
Strapi: 3.4.6 (according to package.json file)
Database: SQLite3 v5.0.0
Operating System: Windows 10 Build 19041.804

Current behavior: enter < npm run develop > and receive the error: 'strapi' is not recognized as an internal or external command, operable program or batch file.
Expected behavior: enter < npm run develop > and have the Strapi app run in development mode

I've tried running npx strapi develop as @andrewrjohn suggested and received the error:
[2021-02-15T16:48:04.267Z] debug ⛔️ Server wasn't able to start properly.
[2021-02-15T16:48:04.278Z] error Error: Cannot find module 'bcryptjs'

I just checked my path (based on what was said by @derrickmehaffy ) and found that npm was properly mapped.

What's frustrating is that it was running last night.  I stopped the service and when I went to restart it this morning I got these errors.

I just re-ran npm install in my directory and also separately ran npm i -g strapi@beta at the suggestion of @shrutisahatpure so I'm not sure which method resolved the issue.  I tried stopping the service again and re-running so I'm hoping it's the updated global install.  There are some real problems with Windows (I've also had trouble installing GraphQl).
